<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qmpPlistWindow</class>
 <widget class="QDialog" name="qmpPlistWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>321</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>321</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>321</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Playlist</string>
  </property>
  <widget class="QPushButton" name="pbAdd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>121</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:left</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/add.png</normaloff>:/img/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddFolder">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>121</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:left</string>
   </property>
   <property name="text">
    <string>Add Folder</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/addfolder.png</normaloff>:/img/addfolder.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRepeat">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>121</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:left</string>
   </property>
   <property name="text">
    <string>Repeat Off</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/repeat-non.png</normaloff>:/img/repeat-non.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pbShuffle">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>121</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:left</string>
   </property>
   <property name="text">
    <string>Shuffle Off</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/shuffle-off.png</normaloff>:/img/shuffle-off.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSave">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>121</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:left</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLoad">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>121</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:left</string>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/load.png</normaloff>:/img/load.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRemove">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>121</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:left</string>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/remove.png</normaloff>:/img/remove.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClear">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>121</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:left</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/clear.png</normaloff>:/img/clear.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="lwFiles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>221</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::InternalMove</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
